circle.yml

```diff
@@ -14,9 +15,6 @@ dependencies:
     - ./DoCIF/commands/buildbaseimage.sh
       # Actually the test step
     - ./DoCIF/commands/runtests.sh
-  cache_directories:
-      # This shouldn't actually cache anything because config.docif says not to
-    - ~/.ccache # See $CACHE_DIRECTORIES in config.docif
```

Comment:
So we have plans to cache things eventually. I understand this is not needed now, but I don't really see a reason to remove it :x